content/webapp/utils/iiif/v3/index.ts

```diff
@@ -349,16 +374,11 @@ type checkModalParams = {
 
 export function checkModalRequired(params: checkModalParams): boolean {
   const { role, auth, isAnyImageOpen, authV2 } = params;
-  // If authV2 is true, We try to use the iiif auth V2 services and fallback to V1 in case the manifest doesn't contain V2
-  const externalAccessService = authV2
-    ? auth?.v2.externalAccessService || auth?.v1.externalAccessService
-    : auth?.v1.externalAccessService;
-  const activeAccessService = authV2
-    ? auth?.v2.activeAccessService || auth?.v1.activeAccessService
-    : auth?.v1.activeAccessService;
-  if (activeAccessService) {
+  const authServices = getAuthServices({ auth, authV2 });
+  if (authServices?.active) {
     return true;
-  } else if (externalAccessService) {
+  } else if (authServices?.external) {
+    // TODO if we've got a v1 service then display a message to say the manifest needs regenerating
```

Comment:
np; we've switched tacks on TODOs when we did the PR standards (see "Address TODOs" in [Pull request process](https://app.gitbook.com/o/-LumfFcEMKx4gYXKAZTQ/s/DPDDj27NI2F2kPukWrC1/readme/github-process/pull-request-process)), so this should either get addressed in this PR or we just ensure there's a ticket for it that you can refer to in this PR as a comment on here (not in the code).

Reply:
Sorry, yep I didn't mean to leave that in. It's related to https://github.com/wellcomecollection/wellcomecollection.org/issues/11257 so I'll add it to that